openlibrary/core/helpers.py

```diff
@@ -109,9 +109,11 @@ def safesort(iterable, key=None, reverse=False):
     care to make that work.
     """
     key = key or (lambda x: x)
+
```

Comment:
These code reformatting changes distract the reviewer when trying to understand the changes.   I would be happy to see the whole codebase formatted with `psf/black --skip-string-normalization` in a ___separate___ PR but let's not do it across multiple PRs.  So, please only make formatting changes in this PR if the on the lines that this PR is modifying.

Reply:
Ok, sure, will do.